src/java/org/apache/cassandra/cql3/QueryHandler.java

```diff
@@ -58,6 +58,9 @@ ResultMessage processBatch(BatchStatement statement,
 
     public static class Prepared
     {
+        // CASSANDRA-19986 Precomputed cache entry size might be available
+        public int cacheEntrySize = -1;
```

Comment:
@bereng do you think this might have sense to make `volatile`?

Reply:
Right you are!